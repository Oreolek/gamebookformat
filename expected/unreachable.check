examples/unreachable.debug: Could not reach section 'nothere' from start.
examples/unreachable.debug: Could not reach section 'nothereeither' from start.
examples/unreachable.debug: Could not reach section 'secondfake' from start.
