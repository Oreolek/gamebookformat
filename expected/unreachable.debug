BEGIN DEBUG OUTPUT
Book title: Gamebook
Number of sections: 6

Turn to 1 to begin.
1 (start) -  It starts here. You can get to 6. 
2 (secondfake) -  Another fake section that is also ignored. 
3 (nothereeither) -  You can not get here either, but because it is tagged as fake there will be no warning generated. 
4 () - (EMPTY)
5 (nothere) -  You can not get here. Verification should warn you about that. 
6 (here) -  You can get here. Congratulations! 
END DEBUG OUTPUT
