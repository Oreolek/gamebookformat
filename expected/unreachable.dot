digraph gamebook {
 1->2
    }
