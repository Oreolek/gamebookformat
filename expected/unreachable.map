{"notevenincluded": 4, "nothere": 5, "here": 6, "start": 1, "nothereeither": 3, "secondfake": 2}