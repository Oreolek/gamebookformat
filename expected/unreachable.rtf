{\rtf1\ansi\ansicpg1252\cocoartf1038\cocoasubrtf360
{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
\paperw11900\paperh16840\margl1440\margr1440\vieww14140\viewh14860\viewkind0
\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\ql\qnatural\pardirnatural

\f0\b\fs24 \cf0

\b \qc Gamebook
\b0\

\b Turn to 1 to begin.
\b0\
\
\b \qc 1
\b0\
\ql  It starts here. You can get to \b 2
\b0
.  \
\

\b \qc 2
\b0\
\ql  You can get here. Congratulations!  \
\

\b \qc 3
\b0\
\ql  Another fake section that is also ignored.  \
\

\b \qc 4
\b0\
\ql  You can not get here. Verification should warn you about that, if enabled (with the --verify option).  \
\

\b \qc 5
\b0\
\ql  You can not get here either, but because it is tagged as fake there will be no warning generated.  \
\

}
