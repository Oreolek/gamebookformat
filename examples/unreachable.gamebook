* 1 start
It starts here. You can get to [[here]].

* here
You can get here. Congratulations!

* nothere
You can not get here. Verification
should warn you about that.

* notevenincluded                                                     :dummy:
This section is not even included in the output. Definitely
ignored when verifying. The section numbering is affected though.
That might be a bug.

* nothereeither                                                        :fake:
You can not get here either, but because it is tagged as
fake there will be no warning generated.

* secondfake                                                           :fake:
Another fake section that is also ignored.

