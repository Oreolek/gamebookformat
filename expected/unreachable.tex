\documentclass[a5,onecolumn]{book}

\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage[hidelinks]{hyperref}
\usepackage{graphicx}

\usepackage[top=3.3cm, bottom=3.3cm, left=2cm, right=2cm]{geometry}
\newif\ifpdf
\ifx\pdfoutput\undefined
  \pdffalse
\else
  \ifnum\pdfoutput=1
    \pdftrue
  \else
    \pdffalse
  \fi
\fi

\title{Gamebook}
\author{}
\date{}

\newcounter{sectionnr}

\begin{document}

\maketitle

\thispagestyle{empty}

\pagestyle{empty}

\clearpage



Turn to 1 to begin.
\phantomsection
\refstepcounter{sectionnr}
\label{section1}
\subsection*{\begin{center} \textbf{1} \end{center}}

 \noindent
 It starts here. You can get to \textbf{\autoref{section2}}. 
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section2}
\subsection*{\begin{center} \textbf{2} \end{center}}

 \noindent
 You can get here. Congratulations! 
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section3}
\subsection*{\begin{center} \textbf{3} \end{center}}

 \noindent
 Another fake section that is also ignored. 
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section4}
\subsection*{\begin{center} \textbf{4} \end{center}}

 \noindent
 You can not get here. Verification should warn you about that, if enabled (with the --verify option). 
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section5}
\subsection*{\begin{center} \textbf{5} \end{center}}

 \noindent
 You can not get here either, but because it is tagged as fake there will be no warning generated. 
\vspace{1em}
\end{document}
